service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ValidateOrderControllerTest.java

```diff
@@ -49,58 +48,81 @@
 
     @Test
     void shouldReturnErrorsWhenTheDateOfIssueIsInFuture() {
-        CaseDetails caseDetails = createCaseDetails(PREVENT_REMOVAL, time.now().plusDays(1),
-            NAMED_DATE, ORDER_TYPE_AND_DOCUMENT, time.now());
+        CaseDetails caseDetails = createCaseDetails(PREVENT_REMOVAL, time.now().plusDays(1), ORDER_TYPE_AND_DOCUMENT);
```

Comment:
minor for most of the test changes you have made here but not something you have done specifically. I do not believe you need to autowire in Time to this test, the base class `AbstractControllerTest` has it and also some convenience methods to get the time in LocalDate or LocalDateTime format